test/FsAutoComplete.Tests.Lsp/Helpers.fs

```diff
@@ -189,6 +189,11 @@ let waitForWorkspaceFinishedParsing (event : Event<string * obj>) =
     if o.Content.Contains """{"Kind":"error","""
     then failtestf "error loading project: %A" o
 
+let mutable projectOptsList : FSharp.Compiler.SourceCodeServices.FSharpProjectOptions list = []
```

Comment:
this mutable will only be valid if we run tests sequentially, right? Do we run in parallel at all? Even if not, I'd add a comment here marking this as an area that would break under parallel tests.